install.sh

```diff
@@ -51,24 +79,29 @@ if ! timeout "$WAIT_TIMEOUT" bash -c \
   die "Ollama startup failed"
 fi
 
-log "📦  Ensuring model is pulled: ${MODEL_ALIAS}..."
+log "📦  Ensuring model is pulled: ${MODEL_ALIAS}…"
 $ENGINE exec ollama-server ollama pull "${MODEL_ALIAS}" > /dev/null 2>&1
 
 ###############################################################################
 # 2. Llama‑Stack
 ###############################################################################
+if [ "$ENGINE" = "docker" ]; then
+  NET_OPTS=( -p "${PORT}:${PORT}" --add-host="${HOST_DNS}:host-gateway" )
+elif [ "$ENGINE" = "podman" ]; then
+  NET_OPTS=( --network host )
+fi
+
+cmd=( run -d "${PLATFORM_OPTS[@]}" --name llama-stack "${NET_OPTS[@]}" \
+      "${SERVER_IMAGE}" --port "${PORT}" \
+      --env INFERENCE_MODEL="${MODEL_ALIAS}" \
+      --env OLLAMA_URL="http://${HOST_DNS}:${OLLAMA_PORT}" )
```

Comment:
HOST_DNS should be `localhost` for macOS. See [comment](https://github.com/meta-llama/llama-stack/pull/2068/files#r2070487610)